### DISCUSSION TITLE
How to add binary to path?

### DISCUSSION THREAD

**Discussion by IdanAdar** [holzschu/a-shell#816] [Q&A]
I am trying to get the IBM Cloud CLI work in a-shell. For that I am experimenting downloading the various binaries and see if any of them work… from here: https://github.com/IBM-Cloud/ibm-cloud-cli-release/releases/

Next, I think I need to either put the binary in usr/bin, but I can’t find it, or add it to the path, but I’m not sure how…
@holzschu could you provide some tips?

**Comment by holzschu** [comment]
There are no ways to add a binary to an existing iOS app. The security systems prevent it.
To be precise, an application can only open a binary if the following conditions are met:
- it is signed with the developer certificate,
- and it is located inside `$APPDIR`, a directory that can only be accessed by Apple.
